### ISSUE TITLE
Dynamic growth of corpus

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#473]
We should think about ways in which we can support corpus creation over time (e.g. in the field methods context). Right now, people just have the option of increasing their raw text files and then re-uploading them. It might be useful to be able to "add" to a corpus (especially across formats -- e.g., create a corpus from ILGs and then add to it from a set of TextGrids). Food for thought.

**Comment by kchall** [comment]
-make sure you can import from OLD files
-ability to generally merge corpora that have been created (regardless of source)
@mmcauliffe this seems like it would be relevant to the kinds of issues you're currently working on

**Comment by mmcauliffe** [comment]
Yeah, at the moment, the ability to add a discourse to a corpus is available in the Python API, just not in the GUI, so all that would be necessary for it would be to write an interface, and large parts of that could be borrow from the existing import corpus GUI.

Is field methods still using OLDs?  They haven't shifted over to LingSync?  I assume that the output formats should be similar across the two, but I'm not sure.
